test/cql-pytest/random_util.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+##################################################################
+
+# Various utility functions which are useful for multiple tests.
+# Note that fixtures aren't here - they are in conftest.py.
+
+# Various utility functions to generate random elements. Using
+# randomized elements allow to have great coverage with time.
+#
+# Use fixture `randomizer_seed` present in conftest.py, to print
+# seed used for generation. If a test fails, this seed can be used to
+# reproduce the test.
+
+import random
+from contextlib import contextmanager
+from util import new_type, unique_name, new_test_table, new_test_keyspace
+from pytest import fixture
+
+# When generating random type if depth > max_depth
+# then force native_type
+max_depth = 2
+# How many types can be in collection/tuple/UDT
+max_types = 5
```

Comment:
I am not very happy about this new file.
In general I am really afraid of what the proliferation of utility functions does to the quality of test suites. In dtest we have a huge layer of convenience functions that add anything but convenience to people who read the tests. Worse - since nobody can even keep track of these convenience functions, they start writing duplicate functions doing similar things. Even when they find an existing function, because it was written for JUST ONE test case, the function is not good enough so they create another one.
Here for example, you have "max_types=5". Why? What if another test wants to use 6? I'm afraid that other test author will simply create yet another set of convenience functions.

My personal preference (but I'm open to be convinced otherwise) is that all this code should NOT be in a separate file if it is only used in one test file. It should be in the the beginning of the same test file where it is used. Only if later, we want to use this code from multiple test files, should we **then** consider moving it to a spearate file. At that point we'll also know if it's really useful code for multiple tests or just a single-purpose hack for a single test.

Reply:
I agree this file isn't ideal, particularly those configuration variables should be per module, not global.
Mainly, I think the development of randomization module in cql-pytest should be separate issue, let's focus here on describe-statement. 

So to not create randomization module here, I've merge this file to `test_describe.py`.

Reply:
Good. If we ever have more users for the randomization code in different tests, we can split them out from test_describe.py at that point (and also we'll see then whether the specific random code you wrote actually is useful for different tests, or they need something different).